### PR TITLE
feat: allow containers to connect to host machine/dockest inside container support

### LINKED ISSUE #79: allow containers to connect to the host machine
In one of my integration tests, I am starting a webserver (via jest) and check whether a webhook on that webserver is called by a service in a docker container.

There are several options for this: https://stackoverflow.com/questions/24319662/from-inside-of-a-docker-container-how-do-i-connect-to-the-localhost-of-the-mach

It would be awesome if this could be easily achieved with this library!

**Edit:**

Some more thoughts:

On platforms like AWS Codebuild, the code is already built inside a container. For starting additional containers the docker socket is injected into the container. That means if you start a new container and want it to be able to connect to the main "test" container you will have to set up a network between those containers.

### PR DESCRIPTION
**Pull request by n1ru4l**
Closes #79

____________

Currently, the jest helpers are available via `dockest/dist/test-helper`, ideally that would be `dockest/test-helper`. This would require an additional build step.

The network name should be randomly generated (as the creation fails in case there already is a network with the given name). The alternative would be to reuse the existing network.

More information can be retrieved from #79

@erikengervall Let me know how you think bout this. 😊

### CODE REVIEW

**Comment by n1ru4l** [comment]
@erikengervall Any idea why the build fails? everything seems to exit with 0 🤔The build does not fail on github actions or when run locally 😕

**Comment by erikengervall** [comment]
> @erikengervall Any idea why the build fails? everything seems to exit with 0 🤔The build does not fail on github actions or when run locally 😕

No idea why this would fail, especially since everything exits successfully. Hm, could there be some lingering resources that causes issues for Travis?

**Comment by erikengervall** [comment]
Found the issue by looking at the most recent GitHub Actions which points out the error much more clearly and actually aborts the tests prematurely:
https://github.com/n1ru4l/dockest/actions
https://github.com/n1ru4l/dockest/runs/244330013

For Travis it can be found at https://travis-ci.com/erikengervall/dockest/builds/130039439#L318. I wonder why it doesn't exit prematurely 😕

File: src/index.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import isDocker from 'is-docker'
```
Comment (erikengervall) [inline src/index.ts]:
```suggestion
import isDocker from 'is-docker' // eslint-disable-line import/default
```

**Pull request merged**